docs/sources/setup-grafana/installation/redhat-rhel-fedora/index.md

```diff
@@ -100,4 +100,4 @@ Complete the following steps to install Grafana using the standalone binaries:
 
 ## Next steps
 
-Refer to [Start the Grafana server]({{< relref "../start-restart-grafana/" >}}).
+Refer to [Start the Grafana server]({{< relref "../../start-restart-grafana/" >}}).
```

Comment:
```suggestion
Refer to [Start the Grafana server]({{< relref "../../start-restart-grafana" >}}).
```

It's convenient to avoid the trailing slash because that means the link won't break if the destination file is changed between being a page (`page.md`), leaf bundle (`page/index.md`), or a branch bundle (`page/_index.md`).

- https://grafana.com/docs/writers-toolkit/review/doc-validator/errata/#relref-has-trailing-slash